### PR TITLE
guides: install from git not npm

### PR DESCRIPTION
**Pull request by pinheadmz**
Closes https://github.com/bcoin-org/bcoin-org.github.io/issues/154

replace all `npm install bcoin` with `git clone...` instructions instead.

Also updated node and npm requirements in the beginner's guide.

### CODE REVIEW

**Comment by braydonf** [comment]
`guides-markdown/beginners.md` looks like a copy of `bcoin/docs/getting-started.md`, probably could mirror the updates from bcoin there?

**Comment by pinheadmz** [comment]
@braydonf thanks, those updates have been integrated

File: guides-markdown/beginners.md [hunk 1]
```diff
@@ -100,7 +145,7 @@ Bcoin CLI is the prepackaged tool for hitting both the REST and RPC api.
 Available in npm as `bclient`.
 
 ```bash
-$ npm install bclient
+$ npm install -g bclient
```
Comment (braydonf) [inline guides-markdown/beginners.md]: Step not necessary.

Comment (pinheadmz) [reply guides-markdown/beginners.md]: oh yeah! updated.

**Comment by braydonf** [comment]
LGTM

**Pull request merged**